flood_ml/data_utils_test.py

```diff
@@ -0,0 +1,37 @@
+import data_utils
```

Comment:
We'll need to update the GitHub CI to actually run these tests (and linters and so on)
It should look a lot like:
https://github.com/UrbanSystemsLab/climateiq-cnn/blob/main/.github/workflows/python-app.yml#L44

Reply:
Added workflows, and looks like they passed. Can you help confirm that these are set up properly?